manifests/cf-manifest/spec/support/manifest_helpers.rb

```diff
@@ -14,15 +14,15 @@ def terraform_fixture(key)
 
   private
 
-  def load_default_manifest
-    output, error, status = Open3.capture3(
-      [
+  def load_default_manifest(environment = "dev")
+    arg_list = [
         File.expand_path("../../../build_manifest.sh", __FILE__),
         File.expand_path("../../fixtures/terraform/*.yml", __FILE__),
         File.expand_path("../../fixtures/cf-secrets.yml", __FILE__),
         File.expand_path("../../fixtures/cf-ssl-certificates.yml", __FILE__),
-      ].join(' ')
-    )
+        File.expand_path("../../fixtures/cf-#{environment}.yml", __FILE__),
```

Comment:
From meatspace conversation with @mtekel: we're going to change this to test against the real files rather than fixtures.